src/Agent.Sdk/Util/StringUtil.cs

```diff
@@ -200,11 +214,11 @@ private static void EnsureLoaded()
 
                 // Initialize the dictionary.
                 var locStrings = new Dictionary<string, object>();
-                foreach (string cultureName in cultureNames)
+                foreach (var cultureName in cultureNames)
                 {
                     // Merge the strings from the file into the instance dictionary.
-                    string assemblyLocation = Path.GetDirectoryName(Assembly.GetEntryAssembly().Location);
-                    string file = Path.Combine(assemblyLocation, cultureName, "strings.json");
+                    var assemblyLocation = Path.GetDirectoryName(Assembly.GetEntryAssembly().Location);
```

Comment:
Was this on purpose? I thought we generally kept the type if you can't deduce from looking at the righthand side of the operator?

Reply:
I've been told by a few people that we should use `var` for everything. I was doing some additional work in this class originally and must have changed that when I was working there. I can change it back for the sake of the PR, but we should decide design-wise on how we should use `var`